types/react-native/index.d.ts

```diff
@@ -3444,17 +3444,17 @@ export interface FlatListProperties<ItemT> extends ScrollViewProperties {
     /**
      * Rendered when the list is empty.
      */
-    ListEmptyComponent?: React.ComponentClass<any> | null
+    ListEmptyComponent?: React.ComponentClass<any> | React.ReactElement<any> | null
 
     /**
      * Rendered at the very end of the list.
      */
-    ListFooterComponent?: React.ComponentClass<any> | (() => React.ReactElement<any>) | null
+    ListFooterComponent?: React.ComponentClass<any> | React.ReactElement<any> | null
```

Comment:
According to the documentation:
> Can be a React Component Class, a render function, or a rendered element.

It looks like adding `React.ReactElement<any>` to the union is correct, but removing the `(() => React.ReactElement<any>)` from the union may be incorrect.